django_perf_rec/api.py

```diff
@@ -15,15 +17,30 @@
 record_current = local()
 
 
-def record(file_name=None, record_name=None):
+def record(file_name=None, record_name=None, path=None):
     test_details = current_test()
 
-    if file_name is None:
-        file_name = test_details.file_path
-        if file_name.endswith('.py'):
-            file_name = file_name[:-len('.py')] + '.perf.yml'
+    if file_name is not None:
+        warnings.warn(
```

Comment:
I've added a test to check this deprecation warning really is being emitted with http://doc.pytest.org/en/latest/recwarn.html

Also I think we need `stacklevel=2` to help people track down where it's coming from, in case they're using the test case mixin